header.html

```diff
@@ -6,6 +6,21 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Course System</title>
-        <link rel="stylesheet" type="text/css" media="screen" href="css/stylesheet.css" />
+        <link rel="stylesheet" type="text/css" media="screen" href="/css/stylesheet.css" />
         <script src="js/script.js"></script>
-    </head>
\ No newline at end of file
+    </head>
+    <header>
+        <div id='menu'>
+            <ul>
+                <li><a href="/home.php">Home</a></li>
+                <li> | </li>
+                <li><a href="/courses/all_courses.php">Courses</a></li>
+                <li><a href="/courseworks/all_courseworks.php">Courseworks</a></li>
+                <li><a href="/students/all_students.php">Students</a></li>
+                <li><a href="/submissions/all_submissions.php">Submissions</a></li>
+            </ul>
+        </div>
+    </header>
+    <hr>
+
+    <body>
```

Comment:
there is no closing tag for <body>

Reply:
Forgot I left that in...

We should probably have an open <body> tag at the end of the header and then a closing </body> tag at the behinning of the footer so that all the page content is within a body tag, but html isn't very strict about enforcing the correct tags. We can remove that tag for now and sort it out in sprint 2 when we fancy up the UI?

Reply:
This looks like it can just be removed.

Reply:
Done :)